src/startupMessage.js

```diff
@@ -0,0 +1,35 @@
+import { getAwsId } from "./aws.js";
+import { groupsSummary } from "./groups.js";
+
+export const nextstrainAbout = `
+  Nextstrain is an open-source project to harness the scientific and public
+  health potential of pathogen genome data.
+  See https://github.com/nextstrain/nextstrain.org for more.
+`;
+
+export async function startupMessage(server, app, groupsApp) {
+
+  const [awsId] = (await Promise.allSettled([
+    groupsApp ? _false() : getAwsId(), // omit AWS user for groups app
+  ])).map(({status, value, reason}) => value || reason); // eslint-disable-line no-unused-vars
```

Comment:
This can be simplified and `_false()` removed:

```suggestion
  // omit AWS user for groups app
  const awsId = groupsApp ? false : await getAwsId();
```

Reply:
Yeah, I contemplated making this simplification here, but chose to leave the machinery in place for a future where we make multiple async requests for data (e.g. as I did for the cognito info, which has now been dropped). I'll make the simplification now, and restore this code when I add subsequent async calls.

Reply:
`Promise.allSettled()` will also, IIRC, handle bare values, e.g. `false`, just fine. And `new Promise((resolve) => resolve(false))` is more idiomatically spelled `Promise.resolve(false)`.

Reply:
> `Promise.allSettled()` will also, IIRC, handle bare values, e.g. `false`, just fine. 

Cool. Indeed it seems it does. The [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled#parameters) state "An iterable of promises".

Reply:
The behaviour is supported by [the spec](https://tc39.es/ecma262/multipage/control-abstraction-objects.html#sec-promise.allsettled) too, but it's a bit harder to read.

Reply:
Resolved in repush. Now using:
```js
const awsId = await getAwsId();
```